srcpkgs/SweetHome3D/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'SweetHome3D'
+pkgname=SweetHome3D
+version=6.1.2
+revision=1
+wrksrc="${pkgname}-${version}-src"
+only_for_archs="i686 x86_64"
+hostmakedepends="unzip"
+makedepends="openjdk apache-ant"
+depends="virtual?java-runtime shared-mime-info desktop-file-utils libXext libXrender"
+short_desc="Free architectural design application"
+maintainer="1is7ac3 <isaac.qa13@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="www.sweethome3d.com"
+distfiles="${SOURCEFORGE_SITE}/sweethome3d/${pkgname}-${version}-src.zip"
+checksum=43645e34c94bf5b514974428c4a0eefae403fcaf52258cbb9c6fe07d4da8f7db
+
+do_build() {
+	ant application textures furniture examples help java3dLibraries
+}
+
+do_install() {
+	vinstall "${FILESDIR}/${pkgname}.desktop" 755 usr/share/applications
```

Comment:
Value of `${pkgname}` never changes, please use it directly.

Reply:
Please also expand all other occurrences of `${pkgname}` in this and future templates.

Reply:
In wrksrc, distfiles and icon name too.